packs/dripstat/sensors/dripstat_alert_sensor.py

```diff
@@ -0,0 +1,55 @@
+from st2reactor.sensor.base import PollingSensor
+import requests
+
+__all_ = [
+    'DripstatAlertSensor'
+]
+
+BASE_URL = 'https://api.dripstat.com/api/v1'
+
```

Comment:
It makes the code co-operate with other non-blocking code. In this case this is the message queue code and sensor service code which is used in the sensor process.

On top of that, in this case, it makes this code more efficient since it makes outgoing http requests for different apps concurrent.